x/emissions/module/rewards/scores_test.go

```diff
@@ -1804,3 +1804,66 @@ func (s *RewardsTestSuite) TestGenerateReputerScoresWithZeroListeningCoefficient
 	s.Require().NoError(err)
 	s.Require().True(coefficient.Coefficient.Equal(params.EpsilonReputer))
 }
+
+func (s *RewardsTestSuite) TestCalculateTopicInitialEmaScore() {
```

Comment:
Add edge cases for scores length 0 and 1.
This can be turned into a table-type test passing in different sets of scores: the current one, an empty one, one with 1 entry only.

Reply:
agree